app/src/main/java/com/yterletskyi/happyfriend/features/settings/domain/SettingEnum.kt

```diff
@@ -1,5 +1,6 @@
 package com.yterletskyi.happyfriend.features.settings.domain
 
 enum class SettingEnum {
-    MY_WISHLIST
+    MY_WISHLIST,
+    APP_VERSION
```

Comment:
let's always use trailing commas. This is not mandatory but highly recommended. 
See: https://kotlinlang.org/docs/coding-conventions.html#trailing-commas